channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -45,6 +41,9 @@ function _getResponseText(xhr: XMLHttpRequest | IXDomainRequest) {
 }
 
 function _getDefaultAppInsightsChannelConfig(): ISenderConfig {
+    let defaultValue: string;
```

Comment:
Because the "names" get replaced as part of minification this identified that the `undefined` was implicitly being cast to `any`, so to get around this we need to define the actual types.